pkg/kubelet/images/image_gc_manager.go

```diff
@@ -142,16 +167,23 @@ func (im *realImageGCManager) Start() error {
 		}
 	}, 5*time.Minute, wait.NeverStop)
 
+	// Start a goroutine periodically updates image cache.
+	// TODO(random-liu): Merge this with the previous loop.
```

Comment:
I don't think we can easily merge the two loops. Not sure what this TODO is meant for.